source/nut/usr/local/emhttp/plugins/nut/include/nut_footer.php

```diff
@@ -77,17 +77,30 @@ function format_time($seconds) {
   }
 
-  if ($battery !== false) {
-    $battery_runtime = array_key_exists($nut_runtime, $ups_status) ? format_time($ups_status[$nut_runtime]) : "n/a";
-    if ($online && $battery < 100) $icon = "<span id='".($config['FOOTER_STYLE'] == 0 ? "nut_battery" : "")."' class='tooltip-nut ".($config['FOOTER_STYLE'] == 1 ? "$black" : "$green")."' data='[{$nut_name}] Online - Battery is charging'><i class='fa fa-battery-charging'></i>&thinsp;{$battery}&thinsp;%</span>";
-    else if ($online && $battery  == 100) $icon = "<span id='".($config['FOOTER_STYLE'] == 0 ? "nut_battery" : "")."' class='tooltip-nut ".($config['FOOTER_STYLE'] == 1 ? "$black" : "$green")."' data='[{$nut_name}] Online - Battery is full'><i class='fa fa-battery-full'></i>&thinsp;{$battery}&thinsp;%</span>";
-    else if (!$online) $icon = "<span id='".($config['FOOTER_STYLE'] == 0 ? "nut_battery" : "")."' class='tooltip-nut $red' data='[{$nut_name}] Offline - Battery is discharging - Est. $battery_runtime left'><i class='fa fa-battery-discharging'></i>&thinsp;{$battery}&thinsp;%</span>";
-    else $icon = "<span id='".($config['FOOTER_STYLE'] == 0 ? "nut_battery" : "")."' class='tooltip-nut ".($config['FOOTER_STYLE'] == 1 ? "$black" : "$green")."' data='[{$nut_name}] Battery status unknown'><i class='fa fa-battery-discharging'></i>n/a</span>";
+  $battery_runtime = array_key_exists($nut_runtime, $ups_status) ? format_time($ups_status[$nut_runtime]) : "n/a";
+  $css_class = $online['severity'] > 0 ? $nut_msgSeverity[$online['severity']]['css_class'] : ($config['FOOTER_STYLE'] == 1 ? $black : $green);
+  $fa_icon = '';
 
-    $status[0] = $icon;
+  $batteryText = $battery . "&thinsp;%";
+  if ($battery === false) {
+    $batteryText = " n/a";
+    $fa_icon = "fa-battery-empty";
+    $online['fulltext'][] = 'Battery info not available';
+  } else if (is_array($online) && in_array('CHRG', $online['value'])) {
+    $fa_icon = "fa-battery-charging";
+  } else if (is_array($online) && in_array('DISCHRG', $online['value'])) {
+    $fa_icon = "fa-battery-discharging";
+    $online['fulltext'][] = "Est. " . $battery_runtime . " left";
+  } else if (is_array($online) && $online['value']) {
+    $fa_icon = "fa-battery-full";
+    $fa_icon .= in_array('RB', $online['value']) ? ' fa-blink' : '';
   } else {
-    $status[0] = "<span id='nut_battery'class='tooltip-nut' style='margin:0 6px 0 12px' data='[$nut_name] Battery info not available'><i class='fa fa-battery-empty'></i>&thinsp;n/a</span>";
+    $fa_icon = "fa-battery-empty";
+    $online['fulltext'][] = 'Battery status unknown';
   }
+  $tooltipData = '[' . $nut_name . '] ' . implode(' - ', $online['fulltext']);
+
+  $status[0] = "<span id='" . ($config['FOOTER_STYLE'] == 0 ? "nut_battery" : "") . "' class='tooltip-nut " . $css_class . "' data='" . $tooltipData . "'><i class='fa " . $fa_icon . "' style='vertical-align: baseline;'></i>&thinsp;" . $batteryText . "</span>";
```

Comment:
@Peuuuur-Noel Maybe `' data='" . $tooltipData` could be included in the check if `FOOTER_STYLE` is `0` to don't show the tooltip when `Small` is enabled if my comment about why it isn't there makes sense to you, the same applies of course to line [126](https://github.com/SimonFair/NUT-unRAID/pull/12/commits/85442bf43ab11e01ac286936772528221e63291b#diff-7b59abd92a7a47a4d9fcabfd2d767ef7343d90c735559552853921584ae40bf2R126)

Reply:
From the forum:

> This was done on purpose and for two reasons.
When I added this to the plugin I was working on small screens remote and I was constantly hovering over it and it kind of bugged me that this always pops up and the second thing was that it made no sense to me to show a tooltip that shows exactly the same.

I just tested this on my small server screen and the tool-tip really does get in the way, maybe we should disable it in the "small" version so users who don't like it can switch to the "small" version and users who like it can keep the regular "normal" version... so nobody is forced to use/not use the tool-tip.

Reply:
Thanks, I made the change.